code/modules/power/lighting.dm

```diff
@@ -653,3 +653,14 @@
 		force = 5
 		playsound(src.loc, 'sound/effects/Glasshit.ogg', 75, 1)
 		update()
+
+
+/obj/machinery/light/floor
```

Comment:
Those really weren't what I was going for. I wanted lights built into the floor, not just another silly colored plating.